packages/react-core/src/components/HelperText/HelperTextItem.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/HelperText/helper-text';
+import { css } from '@patternfly/react-styles';
+
+export interface HelperTextItemProps extends React.HTMLProps<HTMLDivElement | HTMLLIElement> {
+  /** Content rendered inside the helper text item. */
+  children?: React.ReactNode;
+  /** Additional classes applied to the helper text item. */
+  className?: string;
+  /** Sets the component type of the helper text item. */
+  component?: 'div' | 'li';
+  /** Variant styling of the helper text item. */
+  variant?: 'default' | 'indeterminate' | 'warning' | 'success' | 'invalid';
```

Comment:
maybe call the invalid state error, to be same as the:

validated?: 'success' | 'warning' | 'error' | 'default';

From FormGroup?

Not vetoing here, just thinking out loud, feels like we have at least three words which generated error like components:
- error
- danger
- now invalid :)

Reply:
Yeah it may be more consistent to use `error`. The actual modifier is called `invalid` though. @mcarrano Any thoughts on the naming here?

Reply:
@kmcfaul renamed `pf-m-invalid` to `pf-m-error`, you'll get it in core v4.110.4 - https://github.com/patternfly/patternfly/pull/4140